pgsql/pc_pgsql.c

```diff
@@ -550,6 +551,7 @@ pc_patch_dimensional_serialize(const PCPATCH *patch_in)
 	serpch->pcid = patch->schema->pcid;
```

Comment:
Some code could be factored from all patch types into a new `pc_patch_common_serialize` function. (This function could eventually allow serializing version 0 serpatches. is there any point to it ?)